components/Mypage/enrollmentForm.tsx

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import {
+  Button,
+  FormControl,
+  FormLabel,
+  Input,
+  Select,
+} from '@chakra-ui/react';
+import styled from '@emotion/styled';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+import ERROR_MESSAGES from '@/constants/errorMessages';
+import SEOUL_AREA from '@/constants/seoulArea';
+
+const { CHECK_BUSINESS_NUMBER, CHECK_OWNER_NAME } = ERROR_MESSAGES;
+
+type Inputs = {
```

Comment:
타입 이름이 아쉬운데 `Inputs보다는 inputProps등이 더 나을 것 같아요.

```suggestion
type inputProps = {
```

더 좋은 이름이 있을거같은데 생각이 안나네요..